pom.xml

```diff
@@ -46,6 +46,11 @@
             <artifactId>spring-boot-starter-data-jpa</artifactId>
             <version>2.6.4</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <version>2.8.6</version>
+        </dependency>
```

Comment:
Is this used?

Avoid adding new features in a code review, as a good code review should merge to master, and only features that meet the team's Definition of Done should be merged to master.  If functionality is introduced and not completed, it adds to technical debt, as it's unused code.